store/sqlstore/upgrade.go

```diff
@@ -364,8 +368,22 @@ func UpgradeDatabaseToVersion471(sqlStore SqlStore) {
 	}
 }
 
+func UpgradeDatabaseToVersion472(sqlStore SqlStore) {
+	if shouldPerformUpgrade(sqlStore, VERSION_4_7_1, VERSION_4_7_2) {
+		sqlStore.RemoveIndexIfExists("idx_channels_displayname", "Channels")
```

Comment:
Strictly speaking, I could remove this line (but still migrate the version), but I figured it's better to preserve the upgrade code between releases to avoid confusion ("Why is this missing?")